xapi-profiles-structure.md

```diff
@@ -80,7 +80,180 @@ To assist in accomplishing these two primary goals, Profiles also contain metada
 <a name="using-prof-statements"></a>
 ## 5.0 Using Profiles in Statements
 
-Using an introduced Concept, such as an activity type, verb, attachment usage type, extension, activity, or document resource, can be done freely, provided the defined usage and meaning are adhered to. But a Learning Record Provider can go further, and make sure to adhere to Profile-described Statement Templates and Patterns. Learning Record Providers authoring Statements that conform to matching Profile-described Statement Templates and Patterns SHOULD include the most up-to-date conformant Profile version as a category context activity with id equal to the version's `id` in those Statements, and Statements containing a Profile version as a category context activity MUST conform to any matching Statement Templates and Patterns that Profile version describes.
+[//]: <> (This section may need to be moved to after the Profile Component sections)
```

Comment:
I think adding statement validation against profile , against template, against pattern explanation needed here

Reply:
additional clarification where necessary to describe edge cases, etc.